mathics/core/parser/parser.py

```diff
@@ -236,6 +236,12 @@ def parse_postfix(self, expr1, token, p):
     # Called with one Token and return a Node.
     # Used for prefix operators and brackets.
 
+    def p_PatternTest(self,token):
```

Comment:
This point was confusing for me: the name of the token has nothing to do with the prefix op...